jobrunner/lib/database.py

```diff
@@ -104,6 +160,35 @@ def select_values(itemclass, column, **query_params):
     return [decode_field_values(fields, row)[0] for row in cursor]
 
 
+def get_flag(name):
+    """Get the current running mode in the db."""
+    # Note: fail gracefully if the flags table does not exist
+    # This means we don't need to worry about it in local_run.
+    try:
+        cursor = get_connection().execute(
+            f"SELECT value FROM flags WHERE name = '{name}'"
```

Comment:
Hmm, I was trying to avoid using the "ORM", as I was thinking that you'd get back a Flag object, which you'd then have to deference .value on on, or similar.

But I guess we could define a `Flag` dataclas, an use the ORM querying, but just return the `Flag.value` from `get_flag()`?

Reply:
Yeah, I was thinking that logic could go in `get_flag()` which is already going to have to handle the missing table stuff. I agree it's a bit more bureaucratic, but I worry that if we start talking to the db without going through the ORM layer we're going to make it harder to replace it with something better in future.

Reply:
Have added an ORM object, and an `upsert` ORM method.